### PR TITLE
Feature/readme and licence

### PR DESCRIPTION
**Pull request by landgenoot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+* **Jonathan Raes** - [jonathanraes](https://github.com/jonathanraes)
+* **Rico Tubbing** - [klikooo](https://github.com/klikooo)
+
+See also the list of [contributors](https://github.com/digital-voting-pass/digital-voting-pass-appgit/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the LGPL License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
```
Comment (landgenoot) [inline README.md:44]: Maybe add an expression of gratitude to some of our supervisors?

Comment (wkmeijer) [reply README.md:44]: That would be nice I think, Milvum for all the resources they provided and Pouwelse for his guidance in the process.

**Review by wkmeijer — APPROVED** [review]
Nice addition. Now people should have less problems understanding we're doing. :)
You could add an explicit statement that the digitization of the actual ballot paper is out of scope and that this project forms one step towards the digitization of the entire voting process.

**Comment by landgenoot** [comment]
Done

**Pull request merged**